### ISSUE TITLE
batching requests

### ISSUE THREAD

**Issue by jelmer** [nibdo/bloben-calendar#141]
At the moment, bloben seems to essentially request the entire contents of a given CalDAV collection at once. 

For calendars with tens of thousands of entries, or large entries, this can be heavy on the server and slow things down. It would be great if bloben could send a few requests of perhaps a couple of hundred items each (ideally in an order that would speed up display of items, e.g. new to old?). This would also make things easier on the server & browser, as they wouldn't have to deal with 100Mb+ responses.

**Comment by nibdo** [comment]
Yes, hope it will be more optimized in the future. At first, syncing was on demand outside predefined range (f.e. few months). But this would still require some thoughts how to handle repeated events. 

You can query them directly from DAV server, but this was not the best solution with multiple calendars.
